ding2.profile

```diff
@@ -57,8 +57,6 @@ function ding2_form_alter(&$form, &$form_state, $form_id) {
     // Set default values in ting search form to help aegir/bulk installations.
     if ($form_id == 'ting_admin_ting_settings') {
       $form['ting']['ting_search_url']['#default_value'] = 'http://opensearch.addi.dk/3.0/';
-      $form['ting']['ting_scan_url']['#default_value'] = 'http://openscan.addi.dk/2.0/';
-      $form['ting']['ting_spell_url']['#default_value'] = 'http://openspell.addi.dk/1.2/';
```

Comment:
This removes the default form values but I cannot se that the form fields are removed.

I would expect that new form fields as well as default values for these would be added as well.

Is this intentional?